AirportDepartures.playground/Contents.swift

```diff
@@ -16,9 +16,68 @@ import UIKit
 //: e. Use a `String?` for the Terminal, since it may not be set yet (i.e.: waiting to arrive on time)
 //:
 //: f. Use a class to represent a `DepartureBoard` with a list of departure flights, and the current airport
-
-
-
+enum FlightStatus: String {
+    case scheduled = "Scheduled"
+    case canceled = "Canceled"
+    case enRoute = "En Route"
+    case landed = "Landed"
+    case boarding = "Boarding"
+    case departed = "Departed"
+}
+
+enum Timing: String {
+    case onTime = " - On Time"
+    case delayed = " - Delayed"
+    case early = " - Early"
+}
+
+struct Airport {
+    let name: String
+    let abbreviation: String
+    let location: String
+}
+
+struct Flight {
+    let airline: String
+    let number: String
+    let destination: Airport
+    var terminal: String?
+    var departureTime: String?
+    var status: (FlightStatus, Timing?)
```

Comment:
Very nice use of a tuple here! The only thing I would suggest is possibly naming the elements of the tuple so that it is more clear when you use them later, such as line 63. I can't think of what you would name the first element though, except maybe just flightStatus. So you would have status.flightStatus and status.timing. It's not too hard to follow with just two un-named elements though.